### PR TITLE
feat: display scheduled sessions on scheduler

### LINKED ISSUE #1606: Scheduled sessions should be displayed in the scheduler
**Is your feature request related to a problem? Please describe.**

Currently, the scheduler doesn't display scheduled sessions in the calendar.

**Describe the solution you'd like**

Scheduler should display scheduled sessions appropriately.

**Additional context**

Parent issue: #1580 

Working on implementing this feature.

### PR DESCRIPTION
**Pull request by schedutron**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
This PR adds code to display scheduled sessions on the event scheduler.

Fixes #1606

### CODE REVIEW

**Comment by schedutron** [comment]
The scheduler now displays the scheduled sessions appropriately. I have not color coded them because color coding in fullcalendar happens on the basis of microlocation, not session track. I'm reading more about it and will add the color codes in a later PR. I'm trying to see whether fullcalendar supports color coding in the sessions themselves and not the resources.

screen shot 2018-08-11 at 12 22 26 pm

File: app/routes/events/view/scheduler.js [hunk 1]
```diff
@@ -22,8 +22,42 @@ export default Route.extend({
...
     let eventDetails = this.modelFor('events.view');
 
+    let scheduledSessions = await eventDetails.query('sessions', {
+      include : 'speakers',
+      filter  : scheduledFilterOptions
+    });
+
+    let scheduled = []; // to convert sessions data to fullcalendar's requirements
+    scheduledSessions.forEach(function(session) {
+      session.get('microlocation').then(microlocation => {
```
Comment (CosmicCoder96) [inline app/routes/events/view/scheduler.js]: - The formatting is not consistent with rest of the app. Put `.then` in the next line.
- Add a `.catch()` clause to handle failure while fetching microlocation.

**Comment by srv-twry** [comment]
Apologies for the accidental approval. Didn't see the requested changes. Apart from the formatting issues, everything else looks fine.

**Comment by schedutron** [comment]
I just noticed now that I also missed speaker name in the scheduled sessions. Fixing that as well.

**Comment by schedutron** [comment]
@pradeepgangwar @ritikamotwani @srv-twry @CosmicCoder96 Please review again, I have included microlocations while fetching sessions, so there is no need now to fetch them individually, thereby eliminating promise exception handling :)

**Comment by schedutron** [comment]
Rebased

**Pull request merged**